app/v2/models/request/createAmendHistoricNonFhlUkPropertyAnnualSubmission/HistoricNonFhlAnnualAllowances.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.createAmendHistoricNonFhlUkPropertyAnnualSubmission
+
+import play.api.libs.functional.syntax.{toFunctionalBuilderOps, unlift}
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+
+case class HistoricNonFhlAnnualAllowances(
+    annualInvestmentAllowance: Option[BigDecimal],
+    otherCapitalAllowance: Option[BigDecimal],
```

Comment:
Can we move `otherCapitalAllowance` field below `businessPremisesRenovationAllowance` to keep it consistent with the tech spec. Doesn't functionality but helps with reviewing with tech specs.

Reply:
Sure, there was an inconsistency in ordering between the business spec and the example request so I've updated the specs to be consistent, and will update this to match.

Reply:
Yeah, best to go by the request details section rather than the example but can always check the downstream spec for clarification when in doubt